tools/cli/adminCommands.go

```diff
@@ -298,7 +297,7 @@ func connectToCassandra(c *cli.Context) (gocql.Client, gocql.Session) {
 		Timeout:           10 * time.Second,
 	}
 	if c.Bool(FlagEnableTLS) {
-		clusterConfig.TLS = &auth.TLS{
+		clusterConfig.TLS = &config.TLS{
```

Comment:
Not related to this line. 

CLI gets gocql client through a client factory on L309. Can you also update that to use the registered gocql client?

Reply:
It's already updated -- I updated the client factory to use the reigstered gocql client. LMK if I am wrong.

Reply:
I see. But looks like we can remove this methods from client factory interface? internal cli will also do a client registration, so the internal implementation of this method will be the same as the open source version.

Reply:
yeah good catch. Removed.